themes/digital.gov/src/scss/new/_table-of-contents.scss

```
@@ -1,67 +1,46 @@
 @use "uswds-core" as *;
 
-// Table of Contents
+// Table of Contents for resources/list.html:112 will be replaced with future component work
 
-// Hiding the bullets on the TOC except for the guide pages
-.toc:not(.guide-toc) {
+.toc {
   border-image: linear-gradient(to bottom, #4a50c4, #0076d6, #fff) 1;
   border-style: solid;
+  margin-bottom: units(5);
+  padding: units(2) units("205");
 
   @include u-text("indigo-80");
 
-  ul {
-    @include add-list-reset;
-  }
-
-  a {
-    @include u-padding-y(0.5);
-    @include u-padding-x(0.5);
-    @include u-text("normal");
-    @include u-text("no-underline");
-    @include u-text("indigo-70");
-    @include u-font("sans", "sm");
-    @include u-line-height("sans", 2);
-    @include u-display("block");
-
-    &:hover {
-      @include u-shadow(3);
-    }
-  }
-}
-
-.toc {
-  @include u-margin-bottom(5);
-  @include u-padding-x("205");
-  @include u-padding-y(2);
-
   h4 {
-    @include u-margin(0);
-    @include u-margin-bottom(2);
+    margin: units(0);
```

I appreciate the use of units, but these can simply be zero.

```suggestion
    margin: 0;
```